### PR TITLE
Campaign finder  link fix and cleanup

### LINKED ISSUE #2850: [Campaign Finder] Update to fix missing links on default homepage finder tiles
After home page refactor, the campaign tiles for the initial state on the finder are missing links to their respective campaigns.

### PR DESCRIPTION
**Pull request by weerd**
Resolves #2850

@DFurnes Cleanup for JS and some CSS.

@aaronschachter Update to add the link href correctly based on what we discussed.

### CODE REVIEW

File: lib/modules/dosomething/dosomething_home/dosomething_home.module [hunk 1]
```diff
@@ -47,12 +47,9 @@ function dosomething_home_preprocess_node(&$vars) {
...
 
-    $tiles['link'] = l('<span>' . $title . '</span>', 'node/' . $nid, array(
-      'attributes' => array(
-        'class' => array('full-link')
-      ),
-      'html' => TRUE
-    ));
+
+
+    $tiles['link'] = drupal_get_path_alias('node/' . $nid);
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_home/dosomething_home.module]: looks fine. i suppose could call it `'url'` if `link` suggests its markup... but up to you.

Comment (DFurnes) [inline lib/modules/dosomething/dosomething_home/dosomething_home.module]: +1 renaming to `'url'`.

**Comment by DFurnes** [comment]
Seems fine. :nail_care:

File: lib/modules/dosomething/dosomething_home/dosomething_home.module (hunk 1 above)
Comment (weerd) [inline lib/modules/dosomething/dosomething_home/dosomething_home.module]: good note. I'm updating it to this:

```
$tiles['path'] = 'node/' . $nid;
$tiles['pretty_path'] = drupal_get_path_alias($tiles['path']);
```

**Comment by aaronschachter** [comment]
sure :+1:

**Pull request merged**